rollup.config.js

```diff
@@ -1,27 +1,42 @@
 const typescript = require('rollup-plugin-typescript2')
 const { terser } = require('rollup-plugin-terser')
+const { nodeResolve } = require('@rollup/plugin-node-resolve')
+const commonjs = require('@rollup/plugin-commonjs')
 
 const pkg = require('./package.json')
 
-const makeConfigForOutput = output => ({
+const makeConfigForOutput = (output, plugins = []) => ({
   input: './lib/index.ts',
   output,
   plugins: [
     typescript(),
     terser({
       format: {
         comments: false,
-        ecma: 5
-      }
-    })
-  ]
+        ecma: 5,
+      },
+    }),
+    ...plugins,
+  ],
 })
 
 module.exports = [
   makeConfigForOutput({
     file: pkg.main,
     format: 'umd',
     name: 'contentfulExtension',
-    footer: 'globalThis.contentfulApp = globalThis.contentfulExtension;'
-  })
+    footer: 'globalThis.contentfulApp = globalThis.contentfulExtension;',
+    globals: {
```

Comment:
Are these globals required?

Reply:
Yes, Rollup requires them when building.

Reply:
Docs are so confusing.

As I understood, external does not include dependency into bundle:
https://rollupjs.org/configuration-options/#external
> an Array of module IDs, or regular expressions to match module IDs, that should remain external to the bundle. Can also be just a single ID or regular expression. The matched IDs should be either:

And on `globals`:
https://rollupjs.org/configuration-options/#output-globals
> Specifies id: variableName pairs necessary for external imports in umd/iife bundles.

I will try with external.

Reply:
Result:
```
./lib/index.ts → dist/cf-extension-api.js...
(!) Missing global variable name
Use output.globals to specify browser global variable names corresponding to external modules
contentful-management (guessing 'contentfulManagement')
created dist/cf-extension-api.js in 2.7s

./lib/index.ts → dist/cf-extension-api.bundled.js...
created dist/cf-extension-api.bundled.js in 5.2s
```

Reply:
So, `globals` is used to tell rollup that a specific dependency is stored at `globalThis.xxx`. E.g. if you load `jQuery` it assigns itself to `globalThis.$`. `external` adds a `require('...')` statement to the build which is used by the consumer of App SDK to import contentful management.